### PR TITLE
last run updated to save ids list instead only the last id

### PR DESCRIPTION
**Pull request by yedidyacohenpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixe fetch duplicates, and yml content path.
Test playbook added

Force merge for the the yml context output path fixe.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ The following commands from the Carbon Black Endpoint Standard v2 integration ha
     | --- | --- | --- |
     | URL |  | True |
     | Organization Key | The organization unique key. This is required for all use cases \(and for fetching incidents\). | True |
-    | Custom Api Key (for API version 7) | This Custom API key is required for all use cases and for fetch except the policy use cases. | False |
+    | Custom Api Key (for API) | This Custom API key is required for all use cases and for fetch except the policy use cases. | False |
```
Comment (jbabazadeh) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md:54]:
```suggestion
    | Custom Api Key | This Custom API key is required for all use cases and for fetch except the policy use cases. | False |
```

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md [hunk 2]
```diff
@@ -51,9 +51,9 @@ The following commands from the Carbon Black Endpoint Standard v2 integration ha
...
     | Password | This Custom API key is required for all use cases and for fetch except the policy use cases. | False |
-    | Api Key (Api/Live Response key for API version 1) | This Live Response API key is required only for the policy use cases. | False |
+    | Api Key (Api/Live Response key for API) | This Live Response API key is required only for the policy use cases. | False |
```
Comment (jbabazadeh) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md:56]:
```suggestion
    | Api Key (Api/Live Response key) | This Live Response API key is required only for the policy use cases. | False |
```

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/CarbonBlackEndpointStandardV3.yml [hunk 3]
```diff
@@ -15,13 +15,13 @@ configuration:
   required: true
   section: connect
 - name: custom_credentials
-  display: Custom Api Key (for API version 7)
+  display: Custom Api Key (for API)
```
Comment (jbabazadeh) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/CarbonBlackEndpointStandardV3.yml:18]:
```suggestion
  display: Custom Api Key
```

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/CarbonBlackEndpointStandardV3.yml [hunk 4]
```diff
@@ -15,13 +15,13 @@ configuration:
...
   type: 9
   required: false
   additionalinfo: Also required for fetch
   section: connect
 - name: live_response_credentials
-  display: Api Key (Api/Live Response key for API version 1)
+  display: Api Key (Api/Live Response key for API)
```
Comment (jbabazadeh) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/CarbonBlackEndpointStandardV3.yml:24]:
```suggestion
  display: Api Key (Api/Live Response key)
```

File: Packs/CarbonBlackDefense/ReleaseNotes/4_0_1.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Carbon Black Endpoint Standard v3
+
+- Fixed an issue with the **fetch incidents** duplicates.
```
Comment (jbabazadeh) [inline Packs/CarbonBlackDefense/ReleaseNotes/4_0_1.md:6]:
```suggestion
- Fixed an issue where there was duplication with **fetch incidents**
```

**Review by jbabazadeh — CHANGES_REQUESTED** [review]
great work!
please see my comments

File: Tests/conf.json [hunk 6]
```diff
@@ -4042,9 +4042,9 @@
             "playbookID": "Trend Micro Deep Security - Test"
         },
         {
-            "integrations": "Carbon Black Endpoint Standard",
```
Comment (sapirshuker) [inline Tests/conf.json:4045]: please add another entry for the conf, and add the "Carbon Black Endpoint Standard" integration to skip if needed.

Comment (sapirshuker) [reply Tests/conf.json:4045]: Don't think it is a best practice delete a TPB.

Comment (yedidyacohenpalo) [reply Tests/conf.json]: Done

**Review by jbabazadeh — APPROVED** [review]

File: Packs/CarbonBlackDefense/ReleaseNotes/4_0_1.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Carbon Black Endpoint Standard v3
+
+- Fixed an issue where there was duplication with **fetch incidents**.
```
Comment (jbabazadeh) [inline Packs/CarbonBlackDefense/ReleaseNotes/4_0_1.md:6]:
```suggestion
- fixed an issue where the integration fetched duplicate incidents.
```

**Pull request merged**